easybuild/easyconfigs/m/MPICH/MPICH-4.0.2-GCC-11.3.0.eb

```diff
@@ -0,0 +1,21 @@
+name = 'MPICH'
+version = '4.0.2'
+
+homepage = 'https://www.mpich.org/'
+description = """MPICH is a high-performance and widely portable implementation 
+of the Message Passing Interface (MPI) standard (MPI-1, MPI-2 and MPI-3)."""
+
+toolchain = {'name': 'GCC', 'version': '11.3.0'}
+
+source_urls = ['https://www.mpich.org/static/tarballs/%(version)s']
+sources = [SOURCELOWER_TAR_GZ]
+checksums = ['5a42f1a889d4a2d996c26e48cbf9c595cbf4316c6814f7c181e3320d21dedd42']
+
+configopts = 'FFLAGS="-w -fallow-argument-mismatch -O2" FCFLAGS="-fallow-argument-mismatch" \
+    --with-devices=ch4:ucx --with-ucx=$EBROOTUCX'
+
+dependencies = [
```

Comment:
Additional small style suggestion: we usually put `dependencies` above `configopts`.
Strictly speaking the order doesn't matter, but the keeping the order aligned across easyconfigs makes it significantly easier to parse them for humans.